src/main/java/seedu/taassist/storage/TaAssistStorage.java

```diff
@@ -20,9 +20,10 @@ public interface TaAssistStorage {
 
     /**
      * Returns TaAssist data as a {@link ReadOnlyTaAssist}.
-     *   Returns {@code Optional.empty()} if storage file is not found.
-     * @throws DataConversionException if the data in storage is not in the expected format.
-     * @throws IOException if there was any problem when reading from the storage.
+     * Returns {@code Optional.empty()} if storage file is not found.
+     *
```

Comment:
```suggestion
     *
     * @return {@code Optional} encapsulating a {@code ReadOnlyTaAssist} instance.
```

@return param was not specified here.